package.json

```diff
@@ -11,11 +11,10 @@
     "build-orchidui": "npm run build --prefix packages/@orchidui-vue",
     "build-docs": "npm run build --prefix docs",
     "build-playground": "vite build",
-    "build": "npm run build-orchidui && npm run build-docs && npm run build-playground && node obfuscator.js",
+    "build": "npm run build-playground",
```

Comment:
I think that the new build should also be applied to the vercel build, this may be the reason why the bill is falling?

Reply:
maybe build cache https://vercel.com/docs/deployments/troubleshoot-a-build,

@spacemonkey92 can you share build error in vercel